services/movie.py

```diff
@@ -0,0 +1,36 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> Movie:
+    queryset = Movie.objects.all()
+
+    if not genres_ids and not actors_ids:
+        return queryset
+
+    if genres_ids and actors_ids:
+        return queryset.filter(genres__in=genres_ids, actors__in=actors_ids)
+
+    if genres_ids:
+        return queryset.filter(genres__in=genres_ids)
+
+    if actors_ids:
+        return queryset.filter(actors__in=actors_ids)
+
+    return queryset
```

Comment:
In the `get_movies` function, you are returning the queryset immediately after applying the filter. This could lead to unexpected results if you decide to chain more filters or operations after calling this function. A better approach would be to build up your queryset based on the conditions and return it at the end of the function. This way, you can ensure that all conditions are applied to the queryset before it is returned.